.github/workflows/commit-validator.yml

```
@@ -0,0 +1,37 @@
+name: 'Commit Message Validator'
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+  push:
+    branches:
+      - master
+      - 'frontendApp/*'
+      - 'arduino/*'
+    paths-ignore:
+      - '*/workflows/*'
```

Isn't this unnecessarily excessive? Isn't it enough with:
```suggestion
on: pull_request
```

---

GeneraGenerally, I would like to cover all commit pushes and pull requests.

Would "on: pull_request, push" work?

---

`push` won't work since the script compares the "incoming" commits. It will fail by design if it's triggered on the `master` branch.
Since you are not allowed to push directly to the protected branch, for all intents and purposes `pull_request` is as good as `push`.